src/database.lua

```diff
@@ -0,0 +1,27 @@
+local sql = require 'lsqlite3'
+
+db = nil -- global database handle
+
+local database = {}
+
+database.init = function (ms)
+    local table_inits = { ms.irc_factoids.init
+                        -- , ms.irc_quotegrabs.init
```

Comment:
I think this can be simplified/generalized. Instead of creating this table (or the one on line 19). Why not instead have a loop that goes through every module which has both a `dbinit` and `dbcleanup` function. Then, to create a module that has db-interaction, all one must do is expose those functions.